### PR TITLE
Final Group Capstone Project [Doctors' Appointment]

### PR DESCRIPTION
**Pull request by braincee**
**_Hello 👋_**

----------------------------

In this project, we chose "Doctors' Appointment" as a theme
We set up the repos for the front-end and the back-end.
We Installed the necessary dependencies for each side.
We created the API endpoints.
We created the front-end pages and made them responsive. We styled them using Boostrap and CSS
We made API requests and displayed content based on the response we got.
We created tests for both the front-end and the back-end.
We created the API's documentation using swagger.

This is a link to the Swagger API documentation [here](http://127.0.0.1:3001/api-docs/index.html)

This is a link to the Back-End's Pull Request ▶️ [here](https://github.com/braincee/doctors-appointment-back-end/pull/26)

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,15 +1,37 @@
...
 import Login from './components/Login/Login';
+import Doctors from './components/Doctors/Doctors';
+import BookAppointment from './components/Appointments/BookAppointment';
+import MyAppointments from './components/Appointments/MyAppointments';
+import Navbar from './components/layouts/Navbar';
+import AddDoctor from './components/Doctors/AddDoctor';
+import DeleteDoctor from './components/Doctors/DeleteDoctor';
+import DoctorDetails from './components/Doctors/DoctorDetails';
 
 function App() {
```
Comment (Meltrust) [inline src/App.js:15]: - [ ] When I logout and try to login again with my same user I get incorrectly redirected:

image

Kindly correct this bug 👍

Comment (Meltrust) [inline src/App.js:15]: - [ ] When I try to make an appointment and click the "Book appointment" button, this happens:


image

After that, if I go to the appointments screen, I get a blank screen.

Kindly fix this bug to ensure your correct app functionality 👍

Comment (Meltrust) [inline src/App.js:15]: - [ ] When I go to the "Delete doctors" section, the "Doctors" and "Add Doctor" links dissapear.  The test is still there but its not clickable anymore:

image

Kindly correct this bug.

Comment (Meltrust) [inline src/App.js:15]: - [ ] For some reason, the images are not loading:

image

Kindly correct this bug so your app shows as beautiful. 💪

Comment (Meltrust) [inline src/App.js:15]: - [ ] The app is not responsive enough in small devices:

image

Kindly make the app responsive, as this is a requirement for this project:

image

**Review by Meltrust — CHANGES_REQUESTED** [review]
Hi @braincee,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation, but you are almost there!










#### You are really close to finishing the Microverse program!! Keep it up! 👍👍👍

You can do it

### After implementing the requested changes, please submit another review request. ♻️
_Check the comments under the review._

_I strongly recommend you take them into account as they can improve your final evaluation._





Cheers and Happy coding!👏👏👏

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the previous reviews unless it is requested otherwise._**

------

File: src/App.js (hunk 1 above)
Comment (Bria222) [reply src/App.js:15]: @Meltrust we have updated the changes requested

Comment (Meltrust) [inline src/App.js:15]: - [ ] OPTIONAL: The images are still not loading: 

image

I recommend you fix them so your project looks more beautiful. 💪

Comment (Meltrust) [inline src/App.js:15]: - [ ] OPTIONAL: The project is still not responsive on small devices:

image

I highly suggest you correct this issue so your app runs and shows as intended. 👍

**Review by Meltrust — APPROVED** [review]
_note: Approving because the student left a note that the project has been already presented.  Adding my 2 only suggestions as OPTIONAL.
note to student:  Everything in my past review comments was corrected, good job.  The only nitpicks are the non-loading images and the absence of mobile view._


#### Hi @braincee,


# **Wow, you did it 🎉**
Brilliant
## Thank you for the changes implemented 💪 🥇 ㊗️ 
#### _Unless you want to add more features, go ahead to your final presentation ⏩ ⏩ ⏩_ 
## You are about to finish the Microverse program.  You have come a long way!!!




### Good luck in the software industry!!  I'll see you there. ✨
### **Congratulations!!!!!!**  🎉
applause

### To highlight

- Readme has been corrected✔️ 
- Bugs have been corrected✔️ 
- Beautiful frontend✔️
- Awesome job ✔️  



Cheers and Happy coding!👏👏👏


------

**Pull request merged**